dir.traversal.targets

```diff
@@ -39,13 +39,13 @@
     <MSBuild Targets="$(DefaultBuildAllTarget)"
              Projects="@(Project)"
              Condition="'$(SerializeProjects)'=='true' AND '%(Identity)' != ''"
-             Properties="DefaultBuildAllTarget=$(DefaultBuildAllTarget);BuildAllProjects=true;BuildConfiguration=$(BuildConfiguration)"
+             Properties="DefaultBuildAllTarget=$(DefaultBuildAllTarget);BuildAllProjects=true;BuildConfiguration=$(BuildConfiguration);Performance=$(Performance)"
```

Comment:
??  This seems odd - why does this property need to flow down?

Reply:
Consider that the canonical use of this is for configurations, and to build a tree of projects (such as a solution) for several configurations concurrently.

Reply:
The motivation of this change is to make it possible to run perf projects just as is with `dotnet msbuild /t:RebuildAndTest` without needing to pass the additional `/p:Performance` switch to it --> make it more dev friendly. As we don't want to run perf tests when calling .\build-tests which calls src\tests.builds I added the property.

As we only need this to be set for the test target I will remove the ones for the build. I will also talk with @weshaggard  / @ericstj to verify the change's integrity.

Reply:
Sounds good, if they're OK with it then that's good enough for me.

Reply:
OK, global properties are gone. Added a property switch to config.json which seems like the right way to do it.